### PR TITLE
[various] Disable buttons when mixing, autobuyer or purchase tickets running

### PR DESCRIPTION
**Pull request by alexlyp**
To avoid issues and to help UX a bit, we should be disallowing various features to be run at the same time.

Currently when mixer, autobuyer or ticket purchases are running we disable all the other buttons as well as the send transactions.

I've also included a change to the sidebar running indicator css to remove the animation that has been shown to be really bad for performance.

### CODE REVIEW

File: app/components/SideBar/Logo/Logo.module.css [hunk 1]
```diff
@@ -36,7 +36,9 @@
   background-image: var(--menu-mixer-icon);
   background-size: 16px 16px;
   cursor: pointer;
-  animation: spin 1s linear infinite;
+  /* XXX Need a different animation method due to performance of this
```
Comment (amassarwi) [inline app/components/SideBar/Logo/Logo.module.css:39]: Maybe a different `.gif` ? worth creating a separate issue I would say.

File: app/components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/LEGACY_TicketAutoBuyer/Form.js [hunk 2]
```diff
@@ -31,6 +32,19 @@ const TicketAutoBuyerForm = ({
         {isTicketAutoBuyerEnabled ? (
           <AutoBuyerSwitch enabled onClick={onDisableTicketAutoBuyer} />
         ) : (
+          getRunningIndicator ?
+
```
Comment (amassarwi) [inline app/components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/LEGACY_TicketAutoBuyer/Form.js:36]: extra line

**Review by amassarwi — APPROVED** [review]
LGTM!

File: app/components/SideBar/Logo/Logo.module.css (hunk 1 above)
Comment (alexlyp) [reply app/components/SideBar/Logo/Logo.module.css:39]: Yah I'll leave this then call it out in an issue.  There's this and the pending indicators on transactions on the overview that I've noticed be huge perf hogs.

**Review by amassarwi — APPROVED** [review]

**Review by vctt94 — APPROVED** [review]
tACK

**Pull request merged**